jupyterhub_config.py

```diff
@@ -21,12 +21,17 @@
 # using the DOCKER_SPAWN_CMD environment variable.
 spawn_cmd = os.environ.get('DOCKER_SPAWN_CMD', "start-singleuser.sh")
 c.DockerSpawner.extra_create_kwargs.update({ 'command': spawn_cmd })
+
+#NVIDIA Docker config
+c.DockerSpawner.read_only_volumes = {"nvidia_driver_384.59":"/usr/local/nvidia"}
```

Comment:
You certainly don't want to hard code the version of the NVIDIA driver here.
This also assumes that the host has a working nvidia-docker installation.

Reply:
I just commited this code for other purposes, I thought the pull request was only going to show my first commit. I am reverting it soon.